api/resources/translations/messages-es.properties

```diff
@@ -638,7 +639,7 @@ email.invalid = Dirección de correo electrónico no válida.
 empty = El mensaje esta en blanco, por favor reenvielo. Si continua teniendo problemas, informe a su supervisor.
 enketo.constraint.invalid = Valor no permitido
 enketo.constraint.required = Este campo es obligatorio
-enketo.drawwidget.annotation = anotaci�n
+enketo.drawwidget.annotation = anotación
```

Comment:
Ben, were you able to find a solution for these?

Reply:
Yes before the rebase I had fixed it. 

Here it is now as you see it. 

![Screenshot 2024-08-06 at 18 24 18](https://github.com/user-attachments/assets/39a4fb80-1754-4a7b-a5a7-06139422ddee)

I've pushed and now I see it correctly, I hope you do too. 
![Screenshot 2024-08-06 at 19 04 34](https://github.com/user-attachments/assets/bfb69253-30ee-4d5e-929f-ff130822c76b)